WooCommerce/WooCommerceTests/POS/ViewModels/PointOfSaleDashboardViewModelTests.swift

```diff
@@ -81,6 +80,46 @@ final class PointOfSaleDashboardViewModelTests: XCTestCase {
         XCTAssertEqual(sut.isCartCollapsed, expectedCartCollapsedState)
     }
 
+    func test_isAddMoreDisabled_is_true_when_order_is_syncing_and_payment_state_is_idle() {
```

Comment:
If we look at what `PointOfSaleDashboardViewModel` does, it actually doesn't change `isAddMoreDisabled` state when the order is syncing.

It changes it when `totalsViewModel` changes `paymentState` and `isSyncingOrder`. When we're unit testing `PointOfSaleDashboardViewModel`, we're oblivious at how `totalsViewModel` behaves internally. 

When we write tests for `totalsViewModel`, then we can assert that when `totalsViewModel.startSyncingOrder`, then `paymentState` and/or `isSyncingOrder` changes.